pages/upgrade-guide-2.x.md

```diff
@@ -196,6 +196,32 @@ QUnit.module( "router", {
 ```
 
 
+## Replace `assert.throws( block, string, message )` with `assert.throws( block, regexp, message )`
+
+The overload of `assert.throws()` which expected a block, error string, and assertion message has been removed and will now throw an exception. Use a regular expression instead.
```

Comment:
No worries.

I'm traveling today, but I'll find time to address your other comment when
I'm at the airport. Thanks for your feedback!
On Jul 4, 2016 8:56 AM, "Jörn Zaefferer" <notifications@github.com> wrote:

> In pages/upgrade-guide-2.x.md
> <https://github.com/jquery/qunitjs.com/pull/123#discussion_r69461300>:
>
> > @@ -196,6 +196,32 @@ QUnit.module( "router", {
> >  ```
> >
> >
> > +## Replace `assert.throws( block, string, message )` with `assert.throws( block, regexp, message )`
> > +
> > +The overload of `assert.throws()` which expected a block, error string, and assertion message has been removed and will now throw an exception. Use a regular expression instead.
>
> Sorry, that was my fault. A Chrome extension for navigating files within
> GitHub PRs ended up overlapping part of the diff, the one with "now throw"
> on the right.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jquery/qunitjs.com/pull/123/files/07bc612e1b85e6c9259dd6ecc9dbf8d894d1f905#r69461300>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARWekEAPV_90nC6TSBy9WBKTyFYFNTnks5qSREigaJpZM4JD5ct>
> .
>